bin/kano-settings-onboot

```diff
@@ -429,13 +491,21 @@ if __name__ == '__main__':
         set_to_HDMI(False)
 
     # calculate changes needed
-    changes = compare_and_set_mode() or \
-        compare_and_set_full_range() or \
-        compare_and_set_overscan() or \
-        compare_and_set_optimal_resolution()
+    overrall_changes = False
```

Comment:
This variable is misspelt - should be `overall_changes`. Not too much of a problem because it is defined and anyone reading this would understand what it means but can be a pain for anyone wishing to use the variable later.